charmcraft.yaml

```diff
@@ -60,18 +60,22 @@ provides:
     interface: tracing
     description: |
       Integration to offer other charms the possibility to send traces to Tempo.
-
+  provide-datasource-exchange:
```

Comment:
I'm wondering if we can rename the endpoints to be sth like `logs-datasource-exchange` `metrics-datasource-exchange` ? The way it is right now I think is a bit confusing with the terms `provide` and `require`

Reply:
because we're saying its symmetric so it doesnt matter, but assigning a "provider" and a "requirer" in 2 separate endpoints doesn't really explain why we need 2 endpoints instead of just having 1 endpoint with limit 2.

Reply:
I'll explain in the daily what's going on, but I think it makes sense as it is

Reply:
we've decided to simplify this in the end. Tempo will only be requirer, loki and mimir providers.